pyacq/core/rpc/processspawner.py

```diff
@@ -130,20 +67,30 @@ def __init__(self, name=None, addr="tcp://*:*", qt=False, log_addr=None,
         
         # Spawn new process
         class_name = 'QtRPCServer' if qt else 'RPCServer'
-        args = "addr='%s'" % addr
-        bootstrap = bootstrap_template.format(class_name=class_name, args=args,
-                                              bootstrap_addr=bootstrap_addr,
-                                              loglevel=log_level, qt=str(qt),
-                                              logaddr=log_addr, procname=repr(name))
+        args = {'address': address}
+        bootstrap_conf = dict(
+            class_name=class_name, 
+            args=args,
+            bootstrap_addr=bootstrap_addr.decode(),
+            loglevel=log_level,
+            logaddr=log_addr.decode() if log_addr is not None else None,
+            qt=qt,
+            procname=name,
+        )
+        
+        bootstrap_file = os.path.join(os.path.dirname(__file__), 'bootstrap.py')
```

Comment:
I anderstand that it is simpler to write it like this but sometimes  __file__ leads to problems with frozen package (py2exe, pyinstaller, freeze...) no ?

You didn't like the templated way ?